lib/ansible/modules/group_by.py

```diff
@@ -28,9 +29,27 @@
     type: list
     default: all
     version_added: "2.4"
+attributes:
+    action:
+        support: full
+    async:
+        support: none
+    become:
+        support: none
+    bypass_host_loop:
+        support: full
+    check_mode:
+        support: full
```

Comment:
I could see someone assuming this means --check won't modify the group in inventory...

Reply:
well, check_mode means no changes to the target are really made, this is 'safe' in check mode as it doesn't even make permanent changes to the inventory, but i'll add specific description